ed/csspatches/fill-stroke.json.patch

```diff
@@ -0,0 +1,27 @@
+From 9be95c0f7f61ad2ffd296a9346c552ae05b3f519 Mon Sep 17 00:00:00 2001
+From: Francois Daoust <fd@tidoust.net>
+Date: Thu, 4 Mar 2021 17:00:07 +0100
+Subject: [PATCH] Separate stacked combinators
+
+The parser does not support stacked combinators, see discussion in:
+https://github.com/w3c/reffy/issues/494#issuecomment-790713119
```

Comment:
I'm a bit reluctant to integrate patches that target the parser limitations rather than spec bugs;  or more specifically, I don't like the idea of packaging/releasing wrongly-fixed spec data

Reply:
I'm reluctant too but what alternative do you suggest while this gets sorted out?
Drop the `value` fields? Drop the properties? Drop the files?

Reply:
patch the parser :)

Reply:
I don't think I'll have the bandwidth to look into that in the next few days, but please do :)

Reply:
Could we drop this patch and initially not guarantee that everything is possible to parse?

Reply:
having started to look into patching the parser, I agree that this sounds like the best path forward to get something already useful out sooner rather than later

Reply:
I dropped the patches. Tests ignore the problematic entries for the time being.